resources/config.sample.json

```diff
@@ -37,6 +37,10 @@
     "system_log_facility" : "mail",
     "log_message_prefix" : " SFI: ",
     "check_by_job_id" : true,
-    "pyscicat": null
+    "pyscicat": null,
+    "graylog_log" : false,
+    "graylog_host" : "",
+    "graylog_port" : "",
+    "graylog_facility" : "scicat.ingestor"
```

Comment:
@nitrosx Can it be wrapped into a different group ...? Since it's an external service we are using.

Reply:
Sth like #35